mod/profile_photo.php

```diff
@@ -288,7 +288,7 @@ function profile_photo_crop_ui_head(App $a, Image $image)
 	$height = $image->getHeight();
 
 	if ($width < 175 || $height < 175) {
```

Comment:
I just realized that those values probably need to be raised to 300 as well.

Reply:
Don't think so, as this is for **up**scaling.

Reply:
You are probably right, let's see how this pans out.